horace_core/sqw/coord_transform/@aProjectionBase/aProjectionBase.m

```diff
@@ -108,8 +108,8 @@
     end
     %----------------------------------------------------------------------
     properties(Access=protected)
-        alatt_ = [2*pi,2*pi,2*pi]; %unit-sized lattice vector
-        angdeg_= [90,90,90];
+        alatt_ = []; %unit-sized lattice vector
```

Comment:
Is this no longer a unit-sized lattice vector?

Reply:
yes, probably it is not.